subsys/bluetooth/host/keys.h

```diff
@@ -62,9 +67,11 @@ struct bt_keys {
 #if (defined(CONFIG_BT_KEYS_OVERWRITE_OLDEST))
 	u32_t			aging_counter;
 #endif /* CONFIG_BT_KEYS_OVERWRITE_OLDEST */
+	u8_t                    storage_end[0] __aligned(sizeof(void *));
+	u8_t                    state;
```

Comment:
Why not place this in the beginning of the struct so you don't need to add the `storage_end` member?

Reply:
This was the only way I could get it to be the same size of the storage area. Placing it at the front affected the struct padding inside the storage area and it ended up increasing by 3 bytes.
Now it is 120 bytes before and after.

Reply:
I think you can fix that by adding the __aligned annotation for `storage_start`, so that it aligns with the `enc_size`? It wasn't needed until now since `sizeof(id) + sizeof(addr) == 8`.

Reply:
Yeah, much better. Updated to align storage start. We should make sure to do the same for the BR/EDR PR for keys storage.